package.json

```diff
@@ -10,18 +10,23 @@
     "@types/node": "^12.0.0",
     "@types/react": "^16.9.0",
     "@types/react-dom": "^16.9.0",
+    "@types/react-router-dom": "^5.1.3",
```

Comment:
This should go in the dev section plus all other type definitions to reduce on the size of the build file